extensions/common/context.cc

```diff
@@ -520,5 +534,19 @@ bool getAuditPolicy() {
   return shouldAudit;
 }
 
+bool sanitizeBytes(std::string* buf) {
+  char* start = buf->data();
+  const char* const end = start + buf->length();
+  bool modified = false;
+  while (start < end) {
+    if (flatbuffers::FromUTF8(const_cast<const char**>(&start)) < 0) {
```

Comment:
is it possible that `flatbuffers::FromUTF8` access the bytes beyond the last '\0'?

Reply:
No, you can see that flatbuffers has an identical loop https://github.com/google/flatbuffers/blob/16836ff95a7f112f66a8c5994a508e6bfec010c9/src/idl_parser.cpp#L76